### PR TITLE
Use chain in breakout example

### PR DESCRIPTION
**Pull request by hymm**
# Objective

- We should encourage of the simpler to reason about chain.

## Solution

- Use it in the breakout example

---

## Changelog

- Switch breakout to use `chain` instead of `before` and `after`

### CODE REVIEW

**Review by IceSentry — APPROVED** [review]
I'm not sure if this example needs the comments but the change is definitely worth doing either way

**Comment by mgi388** [comment]
> I'm not sure if this example needs the comments but the change is definitely worth doing either way

I’m new to Bevy and the comments in examples help so I would be for it.

**Comment by IceSentry** [comment]
My point was more that these things should be taught somewhere else like the ecs_guide example. We can't teach the concept of `.chain()` or every other bevy feature in every example that uses it.

**Comment by mgi388** [comment]
> My point was more that these things should be taught somewhere else like the ecs_guide example. We can't teach the concept of `.chain()` or every other bevy feature in every example that uses it.

In fact (as you probably already knew) that example already talks about chain so seems unnecessary for this PR then doesn’t it. Thanks for entertaining my comment.

File: examples/games/breakout.rs [hunk 1]
```diff
@@ -60,15 +60,19 @@ fn main() {
...
-                play_collision_sound.after(check_for_collisions),
-            ),
+                play_collision_sound,
+            )
+                // `chain`ing systems together runs them in order
+                .chain(),
+        )
+        .add_systems(
+            Update,
+            // Omitting `chain``, these can run in parallel
```
Comment (rparrett) [inline examples/games/breakout.rs:73]:
```suggestion
            // Omitting `chain`, these can run in parallel
```

Comment (alice-i-cecile) [reply examples/games/breakout.rs:73]:
```suggestion
```
I don't think this is a helpful comment here TBH.

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
On board with the move to .chain above: that logical ordering is correct and it's a nice clean way to express it. I think we should cut the second comment though.

**Review by mockersf — APPROVED** [review]

**Pull request merged**